gnoland/website/pkgs/doc/ast_to_string_test.go

```diff
@@ -0,0 +1,243 @@
+package doc
+
+import (
+	"go/ast"
+	"testing"
+)
+
+func TestGenerateFuncSignature(t *testing.T) {
+	t.Parallel()
```

Comment:
I would suggest not to do tests in parallel unless we are explicitly looking to test behaviour in a parallel setting.

If we want to run tests faster, we can use `make -j8 test` in the root directory. And in the future we could consider splitting further the `pkgs/...` directories, but better to control test parallelism that way :)